src/AstTypeAnalysis.cpp

```diff
@@ -134,9 +134,9 @@ class AstConstraintAnalysis : public AstVisitor<void> {
 
         // print debug information if desired
         if (debug) {
```

Comment:
Since debug is now a pointer, use comparison with nullptr instead.
```debug != nullptr```

Reply:
Fixed in e16185d10dee7789f65b1c444dce58994ab769ea